apps/android/LiveVideoApp/app/src/main/java/com/twilio/livevideo/app/manager/room/RoomManager.kt

```diff
@@ -65,16 +66,20 @@ class RoomManager @Inject constructor(
         }
     }
 
-    fun disconnect() {
-        handleError(null)
+    fun disconnect(fromUI: Boolean = true) {
```

Comment:
I think this function should only be called from the UI and the parameter can then be removed.

Reply:
Done!